clients/python/rostam/kv.py

```
@@ -275,3 +282,76 @@ def _status_message(status: int, payload: bytes) -> str:
+    """Vector-database operations, reached as ``client.vector.*``.
+
+    Shares the parent client's socket pool and auth. The binary arg layouts are
+    in rostam._vecwire and are differential-tested byte-for-byte against the Go
+    encoders; the JSON-carrying parts (metadata, filter, content) round-trip
+    through a real server.
+    """
+
+    def __init__(self, client: "Rostam"):
+        self._c = client
+
+    def create_collection(self, name: str, dim: int, *, metric: str = "cosine", **cfg: Any) -> None:
```

Ruff (0.16.1)

[error] Undefined name `Any`

(F821)